Databases/VSSTester/Logging/Get-VSSWriter.ps1

```diff
@@ -0,0 +1,24 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+function Get-VSSWriter {
+    [CmdletBinding()]
+    param ()
+
+    $writersText = vssadmin list writers
+    if ($LASTEXITCODE) {
+        Write-Warning $writersText
+        throw "Unable to list vss writers"
+    }
+
+    $lineNumber = 3
```

Comment:
This shouldn't need to be here if we defined it in the `for` loop.

Reply:
Good catch. That was left over from before I turned it into a `for`.